src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -177,6 +178,7 @@ interfaces:
       groups:
       - parameters:
         - name
+        flattening_name: get_book
```

Comment:
having `flattening_name` first might be more natural. Also, calling this group `get_book_1` might be a more sustainable pattern.